binding/angular/projects/porcupine-angular/src/lib/porcupine.service.ts

```diff
@@ -30,7 +30,7 @@ export class PorcupineService implements OnDestroy {
 
   public isLoaded$: Subject<boolean> = new Subject<boolean>();
   public isListening$: Subject<boolean> = new Subject<boolean>();
-  public error$: Subject<string | null> = new Subject<string | null>();
+  public error$: Subject<Error | string | null> = new Subject<Error | string | null>();
```

Comment:
Is this ever going to be just a string anymore or are they all Errors now?

Reply:
processcallback returns string

Reply:
is it a big deal to change that as well?